docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.10.0
+    extra_hosts:
+      - 'calypso.localhost:0.0.0.0'
+    environment:
+      CALYPSO_ENV: 'development'
+      NODE_ENV: 'development'
+      HOST: 'calypso.localhost'
+      PORT: 3000
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/calypso
+      - node_modules_volume:/calypso/node_modules
+      - hooks_volume:/calypso/.git/hooks
+    working_dir: /calypso
+    command: make run
+    tty: true # colors!
```

Comment:
Are you sure, it looks like it's still there: https://docs.docker.com/compose/compose-file/#domainname-hostname-ipc-macaddress-privileged-readonly-restart-shmsize-stdinopen-tty-user-workingdir

I don't see a replacement the directive either.